explainer.md

```diff
@@ -0,0 +1,329 @@
+"Control Option Spacebar" to indicate that they wish to click the element which the screen reader is currently visiting.
+These keypresses would never be passed to the application,
+but would be interpreted by the screen reader,
+which would then call the `accessibilityPerformPress()` function on the element via the API.
+This is then routed back to the DOM as a `click` event by the browser.
+
+### Accessibility node properties
+
+Each node in the accessibility tree may be referred to as an **accessibility node**.
+An accessibility node always has a **role**, indicating its semantic purpose.
+This may be a grouping role,
+indicating that this node merely exists to contain and group other nodes,
+or it may be an interactive role,
+such as `"button"`.
+
+![Accessibility nodes in an accessibility tree, showing roles, names, states and properties](images/a11y-node.jpg)
+
+The user, via assistive technology, may explore the accessibility tree at various levels.
+They may interact with grouping nodes,
+such as a landmark element which helps a user navigate sections of the page,
+or they may interact with interactive nodes,
+such as a button.
+In both of these cases,
+the node will usually need to have a **label** (often referred to as a **name**)
+to indicate the node's purpose in context.
+For example, a button may have a label of "Ok" or "Menu".
+
+Accessibility nodes may also have other properties,
+such as the current **value** (e.g. `"10"` for a range, or `"Jane"` for a text input),
+or **state** information (e.g. `"checked"` for a checkbox, or `"focused"`).
+
+Interactive accessibility nodes may also have certain **actions** which may be performed on them.
+For example, a button may expose a `"press"` action.
+
+These properties and actions are referred to as the *semantics* of a node.
+
+## Existing accessibility APIs
+
+The web has rich support for making applications accessible,
+but only via a *declarative* API.
+
+Native HTML elements are implicitly mapped to accessibility APIs.
+For example, an  `<img>` element will automatically be mapped
+to an accessibility node with a role of `"image"`
+and a label based on the `alt` attribute (if present).
+
+![<img> node translated into an image on the page and an accessibility node](images/img-node.jpg)
+
+Alternatively, [ARIA](https://www.w3.org/TR/wai-aria-1.1/)
+allows developers to annotate elements with attributes to override
+the default role and semantic properties of an element -
+but not to expose any accessible actions.
+
+![<div role=checkbox aria-checked=true> translated into a visual presentation and a DOM node](images/aria-checkbox.jpg)
+
+In either case there's a one-to-one correspondence between a DOM node and a node in the accessibility tree,
+and there is minimal fine-grained control over the semantics of the corresponding accessibility node.
+
+### Native platform accessibility APIs
```

Comment:
When you say "native" I don't know what you mean. Is this section entirely about mobile OS? If so you might rephrase as "### Native mobile (iOS, Android, etc) platform accessibility APIs"